apps/app/pages/admin/event.tsx

```diff
@@ -37,7 +38,7 @@ function CreateEvent() {
       .catch((error) => {
         notifyError(
           "Ocorreu um erro",
-          "Não foi possível obter as localizações"
+          "Não foi possível obter as localizações."
```

Comment:
I think this should belong to another PR. We could discuss this consistency thought.